src/main/java/edu/bu/cs673/stockportfolio/api/login/HomeController.java

```diff
@@ -26,13 +26,30 @@
     private final PortfolioService portfolioService;
     private final ResponseService responseService;
 
+    /**
+     * 
+     * Class contructor that handles the view of the homepage
```

Comment:
How does the constructor handle the view? It doesn't handle anything. It initialized dependencies.